.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+          architecture: x64
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '10.19.0'
```

Comment:
Mildly curious -- are we tied to Node 10.19.0? It seems that 10.19 is a few minor versions behind the latest (which I think I saw was 10.23.x).

Reply:
So 10.19.0 is the version that ships with Ubuntu Focal and thus what it was using on Travis. There was an issue when I tried to bump it to 12 (likely due to package.json / package-lock.json having non v12 compatible things), so I decided to keep things sane to move over, then we can upgrade when we take a look at upgrading JS requirements here.